src/html-to-docx.js

```diff
@@ -126,8 +127,13 @@ async function addFilesToContainer(
     // eslint-disable-next-line no-param-reassign
     footerHTMLString = defaultHTMLString;
   }
+  if (documentOptions.decodeUnicode) {
+    // eslint-disable-next-line no-param-reassign
+    htmlString = decode(htmlString);
```

Comment:
dont you want to do this for footer and header content as well if they are present?